src/web/mjs/connectors/MangaDisk.mjs

```diff
@@ -0,0 +1,37 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+export default class MangaDisk extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'mangadisk';
+        super.label = 'Manga Disk';
+        this.tags = [ 'manga', 'indonesian' ];
+        this.url = 'https://mangadisk.web.id';
+        this.path = '/manga/list-mode/';
+        this.queryPages = 'div#readerarea canvas';
+    }
+
+    async _getPages(chapter) {
+        const uri = new URL(chapter.id, this.url);
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryPages);
+
+        return data.map(hash => {
+            const payload = {
+                url: 'https://img.nesia.my.id/image?id='+hash.attributes["file-id"].value,
+                referer: this.url
```

Comment:
The method `_handleConnectorURI()` already has access to `this.url`, so there is no need to provide it via the payload, or was it meant to be `request.url` instead?

Reply:
You are right. Makes no sense.